features/metal/test/test_kernel_config.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+import helper.utils as utils
+
+
+@pytest.mark.parametrize(
+    "kernel_config_item",
+    [
+        "CONFIG_X86_SGX",
```

Comment:
maybe it would be good to add a string field here to provide a reasoning for why this config is desired (or not desired)

string field would be nicer than a comment imo, can be part of the assertion error message

Reply:
I like the idea. This must be extended to contain the following information 
(config name, reasoning string, architecture)

Reply:
further simplified to 
(key, value, rational, arch) 

value should not be implicitly encoded in function calls imo